docs/performance-tuning.asciidoc

```diff
@@ -175,3 +175,14 @@ Limiting the number of spans that may be recorded will reduce memory usage.
 
 Reducing max spans could result in loss of useful data about what occurred within a request,
 if it is set too low.
+
+Instead of limiting the number of spans, it can be better to drop spans with a very short duration. This will record the spans but discard them if they are very short.
+This can be implemented by providing a span-filter:
```

Comment:
I rephrased this part a little bit - I hope you don't mind 😃 

```suggestion
An alternative to limiting the maximum number of spans can be to drop spans with a very short duration, as those might not be that relevant.

This can be implemented by providing a span-filter:
```

I'd like to move the part about "This will record the spans but discard them if they are very short." into a notice box below the code example - something like this:

```asciidoc
NOTE: Using a span filter does not reduce the load of recording the spans in your application, but merely filters them out before sending them to the APM Server.
This, however, both reduces the amount of storage needed to store the spans in Elasticsearch, and the bandwidth needed to transport the data to the APM Server from the instrumented application.
```

A `NOTE` block will render similar to this:

![image](https://user-images.githubusercontent.com/10602/80137069-f764dc80-85a2-11ea-8dbd-5f094ebcc19a.png)